### PR TITLE
Add merge time frame to burn section

### PR DESCRIPTION
**Pull request by ckoopmann**

### CODE REVIEW

File: src/components/TimeFrameControl.tsx [hunk 1]
```diff
@@ -172,12 +171,7 @@ const TimeFrameControl: FC<Props> = ({
     />
   ) : (
     <div className="flex flex-row flex-wrap items-center lg:gap-x-1">
-      {(version === "all"
```
Comment (alextes) [inline src/components/TimeFrameControl.tsx:175]: Ah great, you've cleaned this up as well 🙏 .

File: src/mainsite/components/BurnCategoryWidget.tsx [hunk 2]
```diff
@@ -415,14 +416,14 @@ const BurnCategoryWidget: FC<Props> = ({ onClickTimeFrame, timeFrame }) => {
...
-      ) : timeFrame === "since_burn" ? (
+      ) : timeFrame === "since_burn" || timeFrame == "since_merge" ? (
         <div
           className={`
             flex min-h-[324px] w-full items-center justify-center
             text-center text-lg text-slateus-200
           `}
         >
-          all time frame unavailable, data sync in progress...
+          time frame unavailable, data sync in progress...
```
Comment (alextes) [inline src/mainsite/components/BurnCategoryWidget.tsx:426]: Can we make this just, "time frame unavailable"?

File: src/mainsite/time-frames.ts [hunk 3]
```diff
@@ -6,22 +6,6 @@ export const timeFramesOld = ["m5", "h1", "d1", "d7", "d30", "all"] as const;
 /** @deprecated use TimeFrame instead, with an API map to convert to/from legacy time frames. */
 export type TimeFrameOld = typeof timeFramesOld[number];
 
-export const timeFramesNoMerge = [
```
Comment (alextes) [inline src/mainsite/time-frames.ts:9]: Amazing 🙌 .

Finally getting rid of those 👏 !

**Review by alextes — CHANGES_REQUESTED** [review]
Tiny changes requested because they're so close to what you've edited but looks great!

File: src/mainsite/time-frames.ts (hunk 3 above)
Comment (ckoopmann) [reply src/mainsite/time-frames.ts:9]: Yes finally we can say "all timeframes are created equal"😅

File: src/components/TimeFrameControl.tsx (hunk 1 above)
Comment (ckoopmann) [reply src/components/TimeFrameControl.tsx:175]: As usual, deleted code is the best code😅

File: .gitignore [hunk 4]
```diff
@@ -1,6 +1,7 @@
 **/.DS_Store
 .DS_Store
 .env*.local
+.env
```
Comment (alextes) [inline .gitignore:4]: I use `.env.local` but this works too I'm guessing.

**Review by alextes — APPROVED** [review]

**Pull request merged**